hudi-common/src/main/java/org/apache/hudi/common/util/ObjectSizeCalculator.java

```diff
@@ -54,6 +54,8 @@
  * @author Attila Szegedi
  */
 public class ObjectSizeCalculator {
+  private static final ThreadLocal<ObjectSizeCalculator> objectSizeCalculator = ThreadLocal.withInitial(() -> new ObjectSizeCalculator(CurrentLayout.SPEC));
```

Comment:
The constant name does not conform to the specification.

Reply:
done